modules/transports/core/nhttp/src/main/java/org/apache/synapse/transport/passthru/TargetRequest.java

```diff
@@ -188,11 +188,11 @@ public void start(NHttpClientConnection conn) throws IOException, HttpException
 		if (o != null && o instanceof TreeMap) {
 			Map _headers = (Map) o;
 			String trpContentType = (String) _headers.get(HTTP.CONTENT_TYPE);
-			if (trpContentType != null && !trpContentType.equals("")) {
-				if (!TargetRequestFactory.isMultipartContent(trpContentType)) {
-					addHeader(HTTP.CONTENT_TYPE, trpContentType);
-				}
-			}
+            if (trpContentType != null && !trpContentType.equals("")) {
```

Comment:
formatting seems to be off

Reply:
This is because in original file it has used tabs as spacing.